akka-actor/src/main/scala/akka/routing/MurmurHash.scala

```diff
@@ -8,7 +8,7 @@
 
 /**
  * An implementation of Austin Appleby's MurmurHash 3.0 algorithm
- *  (32 bit version); reference: http://code.google.com/p/smhasher
+ *  (32 bit version); reference: https://code.google.com/p/smhasher
```

Comment:
Perhaps this should now point to https://github.com/aappleby/smhasher ?

Reply:
I know this is redirecting to [https://github.com/aappleby/smhasher](https://github.com/aappleby/smhasher) but I thought it better to keep the original URL. Lemme know if I need to change this as well.

Reply:
I think https://github.com/aappleby/smhasher is a more up-to-date reference so it might be nice to update it here. (the http->https change is already an improvement so I'm also fine with merging as-is)

Reply:
No Problem, Thanks for pointing it out. I have updated the link.